scripts/generate_env_vars.sh

```diff
@@ -33,6 +33,42 @@ function gen_random_string {
     openssl rand -hex 16 | tr -d "\n"
 }
 
+
+function kafka_settings {
+    KAFKA_ROOT_PW=$(gen_random_string)
+    if [ "$AETHER_CONNECT_MODE" = 'CONFLUENT' ]; then
+        cat << EOF1
+KAFKA_URL=${CC_URL}
+KAFKA_SECURITY=SASL_SSL
+# kafka all-tenant Superuser
+KAFKA_SU_USER=${CC_SU_USER}
+KAFKA_SU_PASSWORD=${CC_SU_PASSWORD}
+# default number of replicas to maintain
+KAFKA_REPLICAS=3
+KAFKA_CONSUMER_USER=${CC_SU_USER}
+KAFKA_CONSUMER_PASSWORD=${CC_SU_PASSWORD}
+# # Internal Root User (local only)
+# KAFKA_ROOT_USER=root
+# KAFKA_ROOT_PASSWORD=${KAFKA_ROOT_PW}
```

Comment:
You don't need to comment `KAFKA_ROOT_xxx` variables, the are used only locally so they can stay there and be ignored in CONFLUENT mode, like the zookeeper ones. :wink: